### ISSUE TITLE
Drilldown menus create keyboard trap

### ISSUE THREAD

**Issue by rainbough** [zurb/foundation-sites#9597]
**How can we reproduce this bug?**

1. Step one
Set up a basic drilldown menu using Foundation 6. 
2. Step two
Navigate into the drilldown using keyboard 'tab' for moving down, and 'shift+tab' for moving up.
3. Step three
Notice that you cannot ever navigate out of the drilldown menu using 'tab', 'shirt+tab' or any other keyboard keys.

**What did you expect to happen?**
Keyboard users should be able to navigate out of the menu using the keyboard controls alone. 
**What happened instead?**
Keyboard focus becomes trapped in the drilldown menu. The keyboard user can never navigate out of the menu.

**Test case:**
[CodePen with Foundation 6](http://codepen.io/rainbough/pen/vgEYVp)

Please note this is a level A accessibility requirement. 
[No Keyboard Trap: Understanding SC 2.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html)

I have worked extensively making Foundation 5 off canvas menus keyboard and VO accessible for several projects, but my newer projects are using Foundation 6 with drilldowns inside of the off canvas slide out. I am still familiarizing myself with the Foundation 6 code and have not been able to find a workaround for this issue.

**Comment by kball** [comment]
@Owlbertz can you look into this?

**Comment by Owlbertz** [comment]
Yeah, I will do that.

**Comment by Owlbertz** [comment]
Created a PR for this.
